### PR TITLE
Associations and DivisionIdentifiers

### PR DESCRIPTION
**Pull request by bastrapp**
Added associations to TournamentFormats and DivisionIdentifiers to DivisionRegistrations. Had some problems dropping the unnamed uniqueConstraint in DivisionRegistrations. Could not be deleted in the test environment and caused the tests to fail. So I left it untouched and created a new one that covers the old columns plus the new divisionIdentifier

### CODE REVIEW

File: backend/src/main/resources/database/db.changelog-1.0.xml [hunk 1]
```diff
@@ -643,4 +643,30 @@
...
+		</addColumn>
+		<addForeignKeyConstraint constraintName="fk_tournament_format_association"
+			referencedTableName="ASSOCIATION" baseColumnNames="association"
+			baseTableName="TOURNAMENT_FORMAT" referencedColumnNames="id" />
+		<dropColumn tableName="TOURNAMENT_FORMAT" columnName="dfv_official" />
+	</changeSet>
+	<changeSet author="bas" id="bas43">
+		<addColumn tableName="DIVISION_REGISTRATION">
+			<column name="division_identifier" type="VARCHAR(50)"
+				defaultValue="">
```
Comment (bslbckr) [inline backend/src/main/resources/database/db.changelog-1.0.xml:661]: do we really want to set the empty string as a default value? I would consider not setting a default value at all

File: backend/src/main/resources/database/db.changelog-1.0.xml [hunk 2]
```diff
@@ -643,4 +643,30 @@
...
+				<constraints nullable="false" />
+			</column>
+		</addColumn>
+		<addUniqueConstraint
+			columnNames="tournament_edition, division_type, division_age, division_identifier"
+			tableName="DIVISION_REGISTRATION" constraintName="uk_division_registration_edition_division" />
+	</changeSet>
+	<changeSet author="bas" id="bas44">
+		<tagDatabase tag="v1.2" />
+	</changeSet>
```
Comment (bslbckr) [inline backend/src/main/resources/database/db.changelog-1.0.xml:671]: :+1:

File: backend/src/main/resources/database/db.changelog-1.0.xml (hunk 1 above)
Comment (bastrapp) [inline backend/src/main/resources/database/db.changelog-1.0.xml:661]: I just wanted to make sure the notNull constraint doesn't complain,when inserting nothing...

But how do I change a pull request?

**Comment by bastrapp** [comment]
I found it ;-)

**Pull request merged**